### PR TITLE
Show xrays on the new homepage

### PR DESCRIPTION
**Pull request by ranquild**
Epic: https://github.com/metabase/metabase/issues/21322

How to test:
- Run a fresh Metabase instance. You should see x-rays for the sample database.
- Add another database. You can copy the connection string from the sample database. You should see x-rays for that database and a link to browse it.
- Ignore broken e2e tests for now

Screenshot 2022-03-31 at 20 12 48
Screenshot 2022-03-31 at 20 13 48

### CODE REVIEW

File: frontend/src/metabase/home/homepage/components/HomeContent/HomeContent.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { isSyncCompleted } from "metabase/lib/syncing";
+import { Database } from "metabase-types/api";
+import XraySection from "../../containers/XraySection";
+
+export interface HomeContentProps {
+  databases: Database[];
+}
+
+const HomeContent = ({ databases }: HomeContentProps): JSX.Element | null => {
```
Comment (ranquild) [inline frontend/src/metabase/home/homepage/components/HomeContent/HomeContent.tsx:10]: This was moved to an individual component to not block rendering of `HomeLayout` until the content loads.

File: src/metabase/automagic_dashboards/core.clj [hunk 2]
```diff
@@ -1316,7 +1316,7 @@
                                       first)
                        dashboard (make-dashboard root rule)]
                    {:url         (format "%stable/%s" public-endpoint (u/the-id table))
-                    :title       (:full-name root)
+                    :title       (:short-name root)
```
Comment (ranquild) [inline src/metabase/automagic_dashboards/core.clj:1319]: `full-name` is `Orders table`, but we need only `Orders`

**Review by kdoh — CHANGES_REQUESTED** [review]
At the moment my biggest reaction is finding myself wanting a clearer way to also just go browse my connected database. I know there are some sidebar changes as part of this that are supposed to also make that accessible so let's maybe wait for that but I suspect we'll want an alternate CTA at the bottom of all the X-ray options.

One visual thing: It'd be nice to add a bit of padding at the bottom so when there are more options on a larger set of tables they don't run right into the bottom of the window.
Screen Shot 2022-04-01 at 9 39 03 AM

I also think we might want to tighten up the spacing between the Metabot greeting and the x-rays. There's a bit too much space right now.
Screen Shot 2022-04-01 at 9 40 38 AM

I couldn't tell if it's this branch or not but Metabot's greeting wasn't showing up when I hover on our pal, so that's something to figure out as well.

I also think we should make some copy changes to make it clearer just what is going on. I'll comment on those inline with suggestions.

File: frontend/src/metabase/home/homepage/components/XraySection/XraySection.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  return (
+    <div>
+      {isSample ? (
+        <SectionTitle>
+          {t`Try out these sample x-rays to see what Metabase can do.`}
+        </SectionTitle>
+      ) : (
+        <SectionTitle>
+          {t`Here are some explorations of`}
```
Comment (kdoh) [inline frontend/src/metabase/home/homepage/components/XraySection/XraySection.tsx:42]: I don't have a great alternative yet but I think we might want to give a bit more context for what the heck these are and more actively frame it as a quick way to assist you getting started. Maybe "Get started quickly with some pre-built explorations of your {name} data."?

**Comment by kdoh** [comment]
Ah, I see that the DB is linked to the browse URL right there in the heading, but somehow that didn't click for me the first time I went through.

**Pull request merged**